fineract-provider/src/main/java/org/apache/fineract/portfolio/tax/starter/TaxAutoConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.fineract.portfolio.tax.starter;
+
+import org.apache.fineract.accounting.common.AccountingDropdownReadPlatformService;
+import org.apache.fineract.portfolio.tax.domain.TaxComponentRepository;
+import org.apache.fineract.portfolio.tax.domain.TaxComponentRepositoryWrapper;
+import org.apache.fineract.portfolio.tax.domain.TaxGroupRepository;
+import org.apache.fineract.portfolio.tax.domain.TaxGroupRepositoryWrapper;
+import org.apache.fineract.portfolio.tax.serialization.TaxValidator;
+import org.apache.fineract.portfolio.tax.service.*;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@Configuration
+public class TaxAutoConfiguration {
+
+    @Bean
+    @ConditionalOnMissingBean
```

Comment:
Please provide the Java interface class reference in parenthesis with the `@ConditionalOnMissingBean` annotation.